### DISCUSSION TITLE
Agents - giving worker agents to a manager agent

### DISCUSSION THREAD

**Discussion by deepinderdeol** [microsoft/semantic-kernel#9949] [Q&A] [answered]
Earlier there was a sample showing how you could create a Manager agent and provide it with other agents as its workers. The worker agents could be added as plugins to this manager agent.

Is this pattern still supported? Is there a sample showing this?

**Comment by markwallace-microsoft** [comment, marked as answer]
We have this sample: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Agents/MixedChat_Agents.cs
